Doc/howto/logging-cookbook.rst

```diff
@@ -765,13 +765,71 @@ serialization.
-provides the bare-bones files to run the above functionality using Supervisor: you
-will need to change the ``/path/to/`` parts in the Gist to reflect the actual paths you
-want to use.
-
+To run a logging listener in production, you may need to use a
+process-management tool such as `Supervisor <http://supervisord.org/>`_. `Here
+<https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b>`_ is a Gist
+which provides the bare-bones files to run the above functionality using
+Supervisor. It consists of the following files:
+
++-------------------------+----------------------------------------------------+
+| File                    | Purpose                                            |
++=========================+====================================================+
+| :file:`prepare.sh`      | A Bash script to prepare the environment for       |
+|                         | testing                                            |
++-------------------------+----------------------------------------------------+
+| :file:`supervisor.conf` | The Supervisor configuration file, which has       |
+|                         | entries for the listener and a multi-process web   |
+|                         | application                                        |
++-------------------------+----------------------------------------------------+
+| :file:`ensure_app.sh`   | A Bash script to ensure that Supervisor is running |
+|                         | with the above configuration                       |
++-------------------------+----------------------------------------------------+
+| :file:`log_listener.py` | The socket listener program which receives log     |
+|                         | events and logs them to a file                     |
++-------------------------+----------------------------------------------------+
+| :file:`main.py`         | A simple web application which does logging via a  |
+|                         | socket connected to the listener                   |
++-------------------------+----------------------------------------------------+
+| :file:`webapp.json`     | A JSON configuration file for the web application  |
++-------------------------+----------------------------------------------------+
+| :file:`client.py`       | A Python script to exercise the web application    |
++-------------------------+----------------------------------------------------+
+
+The web application uses `Gunicorn <https://gunicorn.org/>`_, which is a
+popular web application server which starts multiple worker processes to handle
+requests. The whole setup shows how the workers can write to the same log file
+without treading on each other's toes --- they go through the socket listener.
+
+To test these files, do the following in a POSIX environment:
+
+1. Download `the Gist
+   <https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b>`_, using
+   the "Download ZIP" button, as a :file:`zip` archive.
```

Comment:
```suggestion
   <https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b>`_,
   as a ZIP archive using the :guilabel:`Download ZIP` button.
```

* Re-order to simplify the phrasing and make it easier to read/follow
* Use the appropriate role for a GUI button
* Use the proper name rather than a file role